idea/src/META-INF/plugin-common.xml

```diff
@@ -3078,6 +3078,15 @@
                      language="kotlin"
     />
 
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.inspections.BooleanLiteralArgumentInspection"
+                     displayName="Boolean literal arguments"
+                     groupPath="Kotlin"
+                     groupName="Style issues"
+                     enabledByDefault="true"
+                     level="INFO"
```

Comment:
I confess I'm not sure about default level it should have. Will discuss it with my colleagues.